napari/layers/utils/layer_utils.py

```diff
@@ -958,3 +958,18 @@ def _features_to_properties(features: pd.DataFrame) -> Dict[str, np.ndarray]:
     :meth:`_FeatureTable.properties`
     """
     return {name: series.to_numpy() for name, series in features.items()}
+
+
+def _unique_element(array: Array):
```

Comment:
Adding the optional return type helps a little, I think:

```suggestion
def _unique_element(array: Array) -> Optional[Any]:
```